dll/win32/windowscodecs/typeof.h

```diff
@@ -38,7 +38,11 @@ typedef void (__cdecl typeof(jpeg_set_defaults))(struct jpeg_compress_struct *);
 typedef unsigned int (__cdecl typeof(jpeg_write_scanlines))(struct jpeg_compress_struct *, char **, unsigned int);
 
 typedef void (*png_error_ptr_1)(struct png_struct_def *, const char *);
+#ifdef _WIN64
+typedef void (*png_rw_ptr_1)(struct png_struct_def *, unsigned char *, size_t);
```

Comment:
I think this used to be broken but we should now be able to use `size_t` for x86 too.

Reply:
Fixed